plugins/inputs/logfile/tailersrc_test.go

```diff
@@ -304,13 +282,13 @@ func TestOffsetDoneCallBack(t *testing.T) {
 	time.Sleep(2 * time.Second)
 
 	// Removal of log file should stop tailersrc
-	if err := os.Remove(file.Name()); err != nil {
-		t.Errorf("failed to remove log file '%v': %v", file.Name(), err)
-	}
+	err = os.Remove(file.Name())
+	assert.NoError(t, err)
+	time.Sleep(500 * time.Millisecond)
```

Comment:
Why do we need a sleep here? Blocking on `<-done` is supposed to account for this without sleep timers.

Reply:
Ahhhh. I forgot we could use `<-done` to block. Forgot about that. Nice catch